dockerize.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+IMAGE_NAME="boresha/agriculture-core"
+DOCKER_HOME="/agiculture-core/"
+ARG_NUM=2
+
+if [ "$1" == "init" ]; then
+	docker build -t $IMAGE_NAME $DIR
+elif [ "$1" == "build" ]; then
+	docker run --rm -v $(pwd):/agriculture-core/ $IMAGE_NAME npm build
+elif [ "$1" == "run" ]; then
+    docker run --rm --volume "$(pwd):/agriculture-core/" -ti agriculture-core ./start.sh
```

Comment:
Might also be a good idea to add instructions to the README on how to use the dockerize script! (We should also probably consolidate on this since it should be the same commands for the 3 repos we are working on. I am following the way you've done it!)